test/uk/gov/hmrc/agentservicesaccount/controllers/AgentServicesControllerSpec.scala

```diff
@@ -319,136 +316,136 @@ class AgentServicesControllerSpec extends BaseISpec {
-        sixPs.get(1).select("a").get(0).attr("href") shouldBe "https://www.tax.service.gov.uk/send-a-country-by-country-report"
-
-        //      Plastic Packaging Tax
-        val seven = accordion.select("#tax-services-accordion-content-7")
-        seven.select("h4").get(0).text() shouldBe "Before you start"
-        seven.select("h4").get(1).text() shouldBe "Manage your client’s Plastic Packaging Tax"
-
-        val sevenPs = seven.select("p")
-        sevenPs.get(0).text shouldBe "Your client must first register for Plastic Packaging Tax (opens in a new tab)"
-        sevenPs.get(0).select("a").get(0).text shouldBe "register for Plastic Packaging Tax (opens in a new tab)"
-        sevenPs.get(0).select("a").get(0).attr("href") shouldBe "https://www.gov.uk/guidance/register-for-plastic-packaging-tax"
-
-        sevenPs.get(1).text shouldBe "They must then authorise you to act on their behalf"
-        sevenPs.get(1).select("a").get(0).text shouldBe "authorise you to act on their behalf"
-        sevenPs.get(1).select("a").get(0).attr("href") shouldBe "http://localhost:9448/invitations/agents"
-        sevenPs.get(2).text shouldBe "Report your client’s Plastic Packaging Tax and view payments, returns and penalties"
-        sevenPs.get(2).select("a").get(0).text shouldBe "Report your client’s Plastic Packaging Tax and view payments, returns and penalties"
-        sevenPs.get(2).select("a").get(0).attr("href") shouldBe "https://www.tax.service.gov.uk/plastic-packaging-tax/account"
-
-        //Other tax services
-        val eight = accordion.select("#tax-services-accordion-content-8")
-        eight.select(".govuk-warning-text").text shouldBe "! Warning The agent services account is the home for HMRC tax services launched from 2019. For any tax services not listed here, sign out of this account and log in to your HMRC online services for agents account (opens in a new tab)."
-        eight.select(".govuk-warning-text").select("a").get(0).text shouldBe "log in to your HMRC online services for agents account (opens in a new tab)"
-        eight.select(".govuk-warning-text").select("a").get(0).attr("href") shouldBe "https://www.gov.uk/government/collections/hmrc-online-services-for-agents#hmrc-online-services-for-agents-account"
-        // end of accordion
-
-        val helpAndGuidance = html.select("#help-and-guidance-section")
-        helpAndGuidance.select("h2").text shouldBe "Help and guidance"
-        helpAndGuidance.select("p").get(0).text shouldBe "Find out how to use your agent services account and how clients can authorise you to manage their taxes"
-        helpAndGuidance.select("p").get(0).select("a").get(0).text shouldBe "Find out how to use your agent services account and how clients can authorise you to manage their taxes"
-        helpAndGuidance.select("p").get(0).select("a").get(0).attr("href") shouldBe "/agent-services-account/help-and-guidance"
-      }
+//todo double check this with dave and geoff assuming
```

Comment:
Uncomment this / remove it?
We are removing the accordion suspension content (specific to VAT) and then any suspended agent, regardless of regime would be shown the new page for APB-6757... at least that's my understanding but @geoffreywatson can confirm 👀